src/components/App/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Footer from "../Footer";
+import Header from "../Header";
+import Main from "../Main";
+import styles from "./App.module.scss";
+
+function App(props) {
+  const { avatar } = props.userData;
+  return (
+    <React.Fragment>
```

Comment:
you can pre-import Fragment like `import React, { Fragment } from 'react';`
and use just <Fragment></Fragment>